frontend/src/container/NewWidget/index.tsx

```diff
@@ -402,19 +402,20 @@ function NewWidget({ selectedGraph }: NewWidgetProps): JSX.Element {
 
 	return (
 		<Container>
-			<Flex justify="space-between" align="center">
-				<FacingIssueBtn
-					attributes={{
-						uuid: selectedDashboard?.uuid,
-						title: selectedDashboard?.data.title,
-						panelType: graphType,
-						widgetId: query.get('widgetId'),
-						queryType: currentQuery.queryType,
-						screen: 'Dashboard list page',
-					}}
-					eventName="Dashboard: Facing Issues in dashboard"
-					buttonText="Facing Issues in dashboard"
-					message={`Hi Team,
+			<Row>
+				<Col span={8}>
+					<FacingIssueBtn
+						attributes={{
+							uuid: selectedDashboard?.uuid,
+							title: selectedDashboard?.data.title,
+							panelType: graphType,
+							widgetId: query.get('widgetId'),
+							queryType: currentQuery.queryType,
+							screen: 'Dashboard list page',
+						}}
+						eventName="Dashboard: Facing Issues in dashboard"
+						buttonText="Facing Issues in dashboard"
+						message={`Hi Team,
```

Comment:
please move this to a constant. this makes the code unreadable

Reply:
moved all of these to util file